### PR TITLE
Upgrade some fields length

### PR DESCRIPTION
**Pull request by trasher**
I'm also trying to add tests on db update scripts

### CODE REVIEW

File: galette/install/scripts/mysql.sql [hunk 1]
```diff
@@ -7,30 +7,30 @@ DROP TABLE IF EXISTS galette_adherents;
 CREATE TABLE galette_adherents (
   id_adh int(10) unsigned NOT NULL auto_increment,
   id_statut int(10) unsigned NOT NULL default '4',
-  nom_adh varchar(50) NOT NULL default '',
-  prenom_adh varchar(50) NOT NULL default '',
-  pseudo_adh varchar(20) NOT NULL default '',
+  nom_adh varchar(100) NOT NULL default '',
```
Comment (gruiick) [inline galette/install/scripts/mysql.sql]: -> 255

File: galette/install/scripts/mysql.sql [hunk 2]
```diff
@@ -7,30 +7,30 @@ DROP TABLE IF EXISTS galette_adherents;
...
+  prenom_adh varchar(100) NOT NULL default '',
```
Comment (gruiick) [inline galette/install/scripts/mysql.sql]: -> 255

File: galette/install/scripts/mysql.sql [hunk 3]
```diff
@@ -7,30 +7,30 @@ DROP TABLE IF EXISTS galette_adherents;
...
+  pseudo_adh varchar(200) NOT NULL default '',
```
Comment (gruiick) [inline galette/install/scripts/mysql.sql]: -> 255

File: galette/install/scripts/mysql.sql [hunk 4]
```diff
@@ -7,30 +7,30 @@ DROP TABLE IF EXISTS galette_adherents;
...
+  url_adh varchar(255) default NULL,
+  icq_adh varchar(20) default NULL, -- TODO: remove
+  msn_adh varchar(150) default NULL, -- TODO: remove
   jabber_adh varchar(150) default NULL,
   info_adh text,
   info_public_adh text,
   prof_adh varchar(150) default NULL,
-  login_adh varchar(20) NOT NULL default '',
-  mdp_adh varchar(60) NOT NULL default '',
+  login_adh varchar(200) NOT NULL default '',
```
Comment (gruiick) [inline galette/install/scripts/mysql.sql:32]: -> 255

File: galette/install/scripts/mysql.sql [hunk 5]
```diff
@@ -7,30 +7,30 @@ DROP TABLE IF EXISTS galette_adherents;
...
+  mdp_adh varchar(250) NOT NULL default '',
```
Comment (gruiick) [inline galette/install/scripts/mysql.sql:33]: -> 255

File: galette/install/scripts/mysql.sql [hunk 6]
```diff
@@ -41,7 +41,7 @@ CREATE TABLE galette_adherents (
   pref_lang varchar(20) default 'fr_FR',
   lieu_naissance text,
   gpgid text DEFAULT NULL,
-  fingerprint varchar(50) DEFAULT NULL,
+  fingerprint varchar(200) DEFAULT NULL,
```
Comment (gruiick) [inline galette/install/scripts/mysql.sql:44]: -> 255

File: galette/install/scripts/mysql.sql [hunk 7]
```diff
@@ -118,7 +118,7 @@ CREATE TABLE galette_logs (
   id_log int(10) unsigned NOT NULL auto_increment,
   date_log datetime NOT NULL,
   ip_log varchar(46) NOT NULL default '',
-  adh_log varchar(41) NOT NULL default '',
+  adh_log varchar(200) NOT NULL default '', -- see galette_adherents.login_adh
```
Comment (gruiick) [inline galette/install/scripts/mysql.sql:121]: -> 255 (si login_adh passe à 255)

File: galette/install/scripts/pgsql.sql [hunk 8]
```diff
@@ -177,38 +177,38 @@ DROP TABLE IF EXISTS galette_titles CASCADE;
...
 );
 
 DROP TABLE IF EXISTS galette_adherents CASCADE;
 CREATE TABLE galette_adherents (
     id_adh integer DEFAULT nextval('galette_adherents_id_seq'::text) NOT NULL,
     id_statut integer DEFAULT '4' REFERENCES galette_statuts(id_statut) ON DELETE RESTRICT ON UPDATE CASCADE,
-    nom_adh character varying(50) DEFAULT '' NOT NULL,
-    prenom_adh character varying(50) DEFAULT '' NOT NULL,
+    nom_adh character varying(100) DEFAULT '' NOT NULL,
+    prenom_adh character varying(100) DEFAULT '' NOT NULL,
```
Comment (gruiick) [inline galette/install/scripts/pgsql.sql]: -> 255

File: galette/install/scripts/pgsql.sql [hunk 9]
```diff
@@ -177,38 +177,38 @@ DROP TABLE IF EXISTS galette_titles CASCADE;
...
     societe_adh character varying(200) DEFAULT NULL,
-    pseudo_adh character varying(20) DEFAULT '' NOT NULL,
+    pseudo_adh character varying(200) DEFAULT '' NOT NULL,
```
Comment (gruiick) [inline galette/install/scripts/pgsql.sql]: -> 255

File: galette/install/scripts/pgsql.sql [hunk 10]
```diff
@@ -177,38 +177,38 @@ DROP TABLE IF EXISTS galette_titles CASCADE;
...
+    url_adh character varying(255),
+    icq_adh character varying(20), -- TODO: remove
+    msn_adh character varying(150), -- TODO: remove
     jabber_adh character varying(150),
     info_adh text,
     info_public_adh text,
     prof_adh character varying(150),
-    login_adh character varying(20) DEFAULT '' NOT NULL,
-    mdp_adh character varying(60) DEFAULT '' NOT NULL,
+    login_adh character varying(200) DEFAULT '' NOT NULL,
```
Comment (gruiick) [inline galette/install/scripts/pgsql.sql:210]: -> 255

File: galette/install/scripts/pgsql.sql [hunk 11]
```diff
@@ -219,7 +219,7 @@ CREATE TABLE galette_adherents (
     pref_lang character varying(20) DEFAULT 'fr_FR',
     lieu_naissance text DEFAULT '',
     gpgid text DEFAULT NULL,
-    fingerprint character varying(50) DEFAULT NULL,
+    fingerprint character varying(200) DEFAULT NULL,
```
Comment (gruiick) [inline galette/install/scripts/pgsql.sql:222]: -> 255

File: galette/install/scripts/pgsql.sql [hunk 12]
```diff
@@ -274,7 +274,7 @@ CREATE TABLE galette_logs (
   id_log integer DEFAULT nextval('galette_logs_id_seq'::text) NOT NULL,
   date_log timestamp NOT NULL,
   ip_log character varying(46) DEFAULT '' NOT NULL,
-  adh_log character varying(41) DEFAULT '' NOT NULL,
+  adh_log character varying(200) DEFAULT '' NOT NULL, -- see galette_adherents.login_adh
```
Comment (gruiick) [inline galette/install/scripts/pgsql.sql:277]: -> 255

**Comment by trasher** [comment]
Thanks for the feedback @gruiick !

I've applied the fixes you proposed; all lenght changes are in one commit:
https://github.com/galette/galette/pull/98/commits/65c674ff48d4b0ae691ddb148f1e63b12c9d39b4

I'd like you to review some changes I made to fix update process in SQL schemas:
- https://github.com/galette/galette/pull/98/commits/db73691b15020b3699a4f151c5efd7ff0a66e4f3
- https://github.com/galette/galette/pull/98/commits/8a9ac6953f3298e242b0c016cd626653a42d7e7e

Kind regards,
Johan

**Pull request merged**